lib/sufia/arkivo/schema_validator.rb

```diff
@@ -0,0 +1,57 @@
+require 'json-schema'
+
+module Sufia
+  module Arkivo
+    ITEM_SCHEMA = {
+      type: 'object',
+      properties: {
+        token: { type: 'string', required: true },
+        metadata: {
+          type: 'object',
+          required: true,
+          properties: {
+            title: { type: 'string', required: true },
+            creators: { type: 'array', required: true },
+            rights: { type: 'string', required: true },
+            tags: { type: 'array', required: true },
```

Comment:
I don't believe json-schema has the concept of a disjunctive type, unfortunately, no.